geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/serial/SerialWANPropagation_PartitionedRegionDUnitTest.java

```diff
@@ -78,6 +78,41 @@ public void testPartitionedSerialPropagation() {
     vm2.invoke(() -> WANTestBase.validateRegionSize(getTestMethodName() + "_PR", 1000));
   }
 
+  @Test
+  public void testPartitionedSerialPropagationWithTransaction() {
```

Comment:
I would add in the test case name that the senders are not configured in all servers.